### PR TITLE
slim comparisons to sweeps implemented in verification.py

### PR DESCRIPTION
**Pull request by andrewkern**
just verification that i had been doing with slim and have now implemented in the verification.py file.

@gbinux these examples might be of interest

### CODE REVIEW

File: verification.py [hunk 1]
```diff
@@ -878,6 +936,192 @@ def test_sweep_tau_ex3(self):
...
+    population_configurations = [
+        msprime.PopulationConfiguration() for _ in range(slim_ts.num_populations)
+    ]
+
+    recombination_map = msprime.RecombinationMap(
+        positions=[0.0, slim_ts.sequence_length],
+        rates=[r, 0.0],
+        num_loci=int(slim_ts.sequence_length),
+    )
+    recap = msprime.simulate(
```
[the remark this answers is not in the document]
Comment (jeromekelleher) [reply verification.py:952]: If there's multiple populations, I guess you might need
```
demography=msprime.isolated_model([Ne] * slim_ts.num_populations)
```

**Review by jeromekelleher — APPROVED** [review]
LGTM.

**Comment by jeromekelleher** [comment]
@gbinux, can you take a look over this please?

File: verification.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewkern) [reply verification.py:952]: damn i need to get in the habit of using `sim_ancestry`.... will patch this up

**Comment by gbinux** [comment]
> @gbinux, can you take a look over this please?

Sure, I will check this over the weekend.

File: verification.py [hunk 2]
```diff
@@ -878,6 +936,192 @@ def test_sweep_tau_ex3(self):
...
+            v1 = df_slim[stat]
+            v2 = df_msp[stat]
+            sm.graphics.qqplot(v1)
+            sm.qqplot_2samples(v1, v2, line="45")
+            pyplot.xlabel("msp")
+            pyplot.ylabel("SLiM")
+            f = self.output_dir / f"{stat}.png"
+            pyplot.savefig(f, dpi=72)
+            pyplot.close("all")
+            plot_stat_hist(v1, v2, v1.name, v2.name)
```
Comment (gbinux) [inline verification.py:1060]: It looks like v1.name and v2.name are the same (in the figure legend). Maybe directly use 'slim' and 'msp' directly?

File: verification.py [hunk 3]
```diff
@@ -878,6 +936,192 @@ def test_sweep_tau_ex3(self):
...
+            f = self.output_dir / f"{stat}.hist.png"
+            pyplot.savefig(f, dpi=72)
+            pyplot.close("all")
+        pyplot.plot(mids, np.array(msp_win_pis).mean(axis=0), label="msp")
+        pyplot.plot(mids, np.array(slim_win_pis).mean(axis=0), label="slim")
+        pyplot.title(f"tau: {tau}")
```
Comment (gbinux) [inline verification.py:1066]: Another nitpicking point: I would suggest adding the xlabel and ylabel to this plot

File: verification.py [hunk 4]
```diff
@@ -878,6 +936,192 @@ def test_sweep_tau_ex3(self):
...
+        self._run(10, 1e6, 1e3, 1e-7, 0.25, 200, num_replicates=100)
+
+    def test_sweep_vs_slim_ex3(self):
+        self._run(10, 1e6, 1e3, 1e-7, 0.25, 1000, num_replicates=100)
+
+    def test_sweep_vs_slim_ex4(self):
+        self._run(10, 1e6, 1e3, 1e-7, 0.25, 2000, num_replicates=100)
+
+    def test_sweep_vs_slim_ex5(self):
+        self._run(10, 1e6, 1e3, 1e-7, 0.25, 5000, num_replicates=100)
```
Comment (gbinux) [inline verification.py:1122]: On my desktop, it took about 20 minutes to run the five tests. Maybe we can change the parameters a bit to make it run faster?

**Review by gbinux — APPROVED** [review]
This msp vs slim comparison code looks great. I am excited about the generated plots! Thanks!

**Comment by jeromekelleher** [comment]
@andrewkern, can we make the small changes here so we can merge please? Don't want this to get stale.

File: verification.py (hunk 4 above)
Comment (andrewkern) [reply verification.py:1122]: i reduced this to 10 reps but it will be quite noisy now...

**Comment by andrewkern** [comment]
@jeromekelleher-- made all these edits and rebased. should be ready to merge.

**Review by jeromekelleher — APPROVED** [review]
LGTM!

**Pull request merged**